chat/attachment.go

```diff
@@ -33,6 +33,10 @@ type Attachment struct {
 	// Fields contains optional fields to be displayed in the in a table inside the attachment.
 	Fields []*Field `json:"fields,omitempty"`
 
+	// MarkdownIn enables Markdown support. Valid values are ["pretext", "text", "fields"].
+	// Setting "fields" will enable markup formatting for the value of each field.
+	MarkdownIn []string `json:"mrkdwn_in,omitempty"`
```

Comment:
Just spotted this renames the json value lets try to maintain consistency by keeping it the same just converting to camel case as with all the other fields.